lib/rubocop/rspec_rails/description_extractor.rb

```diff
@@ -32,8 +32,8 @@ def initialize(yardoc)
         # Test if the YARD code object documents a concrete rspec cop class
         #
         # @return [Boolean]
-        def rspec_cop?
-          cop_subclass? && !abstract? && rspec_cop_namespace?
+        def rspec_rails_cop?
+          cop_subclass? && !abstract? && rspec_rails_cop_namespace?
```

Comment:
I think the last part is not necessary.
```suggestion
          cop_subclass? && !abstract?
```